ArduPlane/Attitude.cpp

```diff
@@ -572,6 +572,16 @@ bool Plane::suppress_throttle(void)
 
     if (control_mode==AUTO && 
         auto_state.takeoff_complete == false) {
+
+        if (is_flying() &&
+            millis() - started_flying_ms > 5000 && // been flying >5s in any mode
```

Comment:
It takes about 2-3 seconds for is_flying to go true so this is really about an 8 second timer. However, for bungee launches my throttle delay timer is set for 200ms so I'm not sure why 8s, or even 5s, would be anywhere close to being a problem.